agent/service.go

```diff
@@ -154,7 +155,25 @@ func (ts *TaskService) Create(requestCtx context.Context, req *taskAPI.CreateTas
 		return nil, fmt.Errorf("Drive %q could not be found", driveID)
 	}
 
-	err = bundleDir.MountRootfs(drive.Path(), "ext4", nil)
+	const (
+		maxRetries = 100
+		retryDelay = 10 * time.Millisecond
+	)
+
+	// We retry here due to Firecracker needing some time to populate the guest
```

Comment:
nit: I think it's more accurate to say the guest kernel needs time to rescan the guest drive

Reply:
Yes, the asynchronous operation that needs to complete is 100% in the guest kernel. Firecracker is out of the picture at this point.

Reply:
Ah, that's correct. Reworded!